dispatcher/MongoConnect.py

```diff
@@ -203,6 +198,7 @@ def aggregate_status(self):
                     'pll_unlocks': 0,
                     'number': -1}
                 for k in self.dc}
+        phys_stat = {}
         for detector in self.latest_status.keys():
             # detector = logical
             statuses = {}
```

Comment:
For me personally, it's getting a bit hard to follow all the `status`, `statuses` and the `doc['status']` fields are supposed to do and how they differ

Reply:
It's probably clearer than it was before by the way so thanks for cleaning it up 👍 

I'm just mentioning this because a mistake here would not be covered by this current review.

Reply:
`status` is the extracted status of the host currently being evaluated, or the aggregated status of a logical detector. It's largely a short-term variable. `statuses` is the dict of all the extracted statuses so far, and `doc['status']` is whatever's in the document, which is probably unprocessed.